serializations.md

```diff
@@ -562,6 +562,23 @@ The recipient must be one of the following:
 ]
 ```
 
+#### Channel create transaction (version 2, from Iris release)
+```
+[ <initiator>               :: id()
+, <initiator_amount>        :: int()
+, <responder>               :: id()
+, <responder_amount>        :: int()
+, <channel_reserve>         :: int()
+, <lock_period>             :: int()
+, <ttl>                     :: int()
+, <fee>                     :: int()
+, <initiator_delegate_ids>  :: [id()]
```

Comment:
Is this right? See [the code in aesc_channels.erl](https://github.com/aeternity/aeternity/blob/GH-3286_fp_delegates/apps/aechannel/src/aesc_channels.erl#L197-L211)

Reply:
never mind